src/index.html

```diff
@@ -21,7 +21,366 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <a
+        href="index.html"
+        class="header__logo-link"
+      >
+        <img
+          src="images/logo.png"
```

Comment:
Ensure that the `logo.png` image exists in the `images` directory. Incorrect paths or missing images can result in broken visuals, which is crucial for the user interface.